lib/ansible/modules/network/iosxr/iosxr_command.py

```diff
@@ -125,9 +126,9 @@
 import time
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.network.iosxr.iosxr import run_commands, iosxr_argument_spec, check_args
+from ansible.module_utils.network.iosxr.iosxr import run_command, iosxr_argument_spec
+from ansible.module_utils.network.iosxr.iosxr import check_args, command_spec
```

Comment:
check_arg() call can be removed from all iosx module and utils and it is not longer used